### ISSUE TITLE
Drop down role is missing

### ISSUE THREAD

**Issue by rseyum** [rseyum/github-issue-templates#10]
**Describe the bug**
We need to add the role for the drop-down to be announced by a screen reader. this also affects visually disabled people.

**To Reproduce**
1. open the screen reader and navigate to the drop-down
2. Check if it is announced by screen reader
3. go to the drop-down and add role="Combobox"
4. check the result 

**Expected behavior**
The screen reader should announce the dropdown list.

**Screenshots**
N/A

**Desktop (please complete the following information):**
N/A

**Smartphone (please complete the following information):**
N/A

**Additional context**
N/A
